worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -187,7 +188,7 @@ public void save(Actor actor, LocalSession session,
         SchematicSaveTask task = new SchematicSaveTask(actor, f, format, holder, overwrite);
         AsyncCommandBuilder.wrap(task, actor)
                 .registerWithSupervisor(worldEdit.getSupervisor(), "Saving schematic " + filename)
```

Comment:
Don't we need some translations for these other messages as well? Or are we going to do a second pass on them?

Reply:
So IMO at this point, a second pass makes the most sense, to try and get the base translations infrastructure in as soon as possible.

Reply:
I completely agree with Matt on this one.